src/github/githubRepository.ts

```diff
@@ -581,7 +581,7 @@ export class GitHubRepository implements vscode.Disposable {
 		}
 	}
 
-	async getPullRequest(id: number): Promise<PullRequestModel | undefined> {
+	async getPullRequest(id: number): Promise<PullRequest | undefined> {

```

Comment:
Why have all these methods changed from returning a PullRequestModel to a PullRequest?

Reply:
This was a consequence of adding the folder manager to the PR - I was delaying creation of the model since I didn't have it here. Have now reverted this